private_acl_rules.tf

```diff
@@ -380,3 +380,28 @@ resource "aws_network_acl_rule" "private_egress_to_cool_via_ipa_ports" {
   from_port      = each.value.port
   to_port        = each.value.port
 }
+
+# Allow ingress from the operations subnet for traffic that is to be
+# relayed to COOL Shared Services via IPA-related ports
+#
+# For: Operations instance communication with FreeIPA
+#
+# Note that these rules only apply to the private subnet with
+# Guacamole.
+resource "aws_network_acl_rule" "private_ingress_from_operations_subnet_via_ipa_ports" {
+  provider = aws.provisionassessment
+  # for_each = local.ipa_ports
```

Comment:
Is there a reason this is not using this `for_each`? The name for this resource does not match what is actually being defined here.